assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -46,7 +46,7 @@ import {
 
 export const TEXT = {
   CONTEXT: __(
-    "The story editor will append a default, configurable AMP analytics configuration to your story. If you're interested in going beyond what the default configuration is, read this article on<a>analytics for your Web Stories</a>.",
```

Comment:
Thanks for the assist here, @swissspidy - I'd eliminated the space between the initial anchor tag and the character before it because 0.25em is getting added within the anchor and it was making the space between text longer. I think you're right to have this update here - the padding should be dropped and links treated the same so we'll catch that as we reskin. Just thought I'd explain my madness. 🙃